code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -449,3 +450,8 @@
 /obj/structure/closet/bluespace/close()
 	. = ..()
 	density = 0
+
+/obj/structure/closet/examine(mob/user)
+	. = ..() 					//This does the standard locker examine proc
+	if(emagged)			//show text that its emagged to user
+		. += "<span class='notice'>The ID lock is sparking, looks like it was hacked.</span>"
```

Comment:
The word `hacked` here seems wrong, since theres a difference between something being hacked and emagged. Maybe replace it with `looks like it has been fried`

Reply:
Sounds better yes, will change that

Reply:
time to update the pictures too then